src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/service/NetworkWatchListService.java

```diff
@@ -104,4 +146,40 @@ public void tick() {
     private long now() {
         return System.nanoTime();
     }
+
+    private void addTimeout(Endpoint endpoint, long key) {
```

Comment:
It will be better if we key will be renamed in something like `timeoutAt`.

Reply:
Renamed.